homeassistant/components/inels/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""The iNels integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from inelsmqtt import InelsMqtt
+from inelsmqtt.discovery import InelsDiscovery
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import BROKER, BROKER_CONFIG, DEVICES, DOMAIN, LOGGER
+
+PLATFORMS: list[Platform] = [
+    Platform.BUTTON,
+    Platform.SWITCH,
+    Platform.LIGHT,
+    Platform.COVER,
+    Platform.SENSOR,
+    Platform.WATER_HEATER,
+    Platform.CLIMATE,
+    Platform.BINARY_SENSOR,
+    Platform.SELECT,
+]
```

Comment:
Please limit the initial PR to a single platform.

See also: <https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
Should I comment out the Platform lines or should I delete the platform files and remove the lines on the platform list?

Reply:
All other platforms should be fully removed.